### PR TITLE
docs: Propose Windows Visual Studio Build Tools

### PR DESCRIPTION
**Pull request by Gustl22**
_Description of what this PR is changing or adding, and why:_

Add alternative to VS without Editor, to only install Build tools.

_Issues fixed by this PR (if any):_
https://github.com/flutter/flutter/issues/77638#issuecomment-1216915377

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by khanhnwin** [comment]
Thanks @Gustl22! PTAL @cbracken

File: src/development/platform-integration/desktop.md
Comment (cbracken) [inline src/development/platform-integration/desktop.md:61]: Since most users will be more comfortable with a graphical IDE to go with their install, and because it's analogous to what we recommend for macOS (a full Xcode install), I'd phrase this something like:

* [Visual Studio 2022][] When installing Visual Studio,
  select the "Desktop development with C++" workload,
  including all of its default components,
  to install the necessary C++ toolchain and
  Windows SDK header files. Alternatively, 
  [Visual Studio Build Tools 2022][] can be installed if
  only the Windows command-line tools are desired.

**Review by cbracken — COMMENTED** [review]
Thanks for the patch! Overall looks good - just a wording suggesting.

**Review by domesticmouse — APPROVED** [review]
LGTM once @cbracken's change is merged

File: src/development/platform-integration/desktop.md
Comment (Gustl22) [reply src/development/platform-integration/desktop.md:61]: This sounds like you don't need to install the C++ workload with Build tools in order to be able to develop in windows.
My proposal:
```
Alternatively, [Visual Studio Build Tools 2022][] 
can be installed with the same workload if
only the Windows command-line tools are desired.
```

Comment (Gustl22) [reply src/development/platform-integration/desktop.md:61]: @cbracken any opinions? Thx :)

**Review by cbracken — APPROVED** [review]

**Review by khanhnwin — APPROVED** [review]

**Comment by khanhnwin** [comment]
This looks ready to land! Thanks @Gustl22 @cbracken!

**Comment by YouJiacheng** [comment]
For developers in China, https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022 will be redirected to https://visualstudio.microsoft.com/zh-hans/downloads/, in which I can't find Build Tools channel or download MSBuild only.

I think https://aka.ms/vs/17/release/vs_buildtools.exe should be included in docs as well, with which I can download an evergreen (online) bootstrapper in one click.
I found this link in https://learn.microsoft.com/en-us/visualstudio/install/create-an-offline-installation-of-visual-studio?view=vs-2022 and https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-history#release-dates-and-build-numbers.

Also, a `winget` alternative can be helpful:
```pwsh
winget install --id=Microsoft.VisualStudio.2022.BuildTools  -e
```

**Pull request merged**
